packages/odf/features.ts

```diff
@@ -94,3 +101,49 @@ export const setOCSFlags = async (setFlag: SetFeatureFlag) => {
   ocsDetector();
   ocsIntervalId = setInterval(ocsDetector, 15 * SECOND);
 };
+
+
+export const detectACMObservability = async (setFeatureFlag: SetFeatureFlag) => {
+  let id = null;
+  let isInitial = true;
+  let mcokind: K8sModel = {...AcmMultiClusterObservabilityModel}
```

Comment:
```suggestion
  let mcokind: K8sModel = AcmMultiClusterObservabilityModel
```
Wouldn't this work?
Same for configMapKind